app/backend/db/schemas.py

```diff
@@ -25,6 +25,11 @@ class Config:
         orm_mode = True
 
 
+class UserAuth(User):
+    username: str
+    password: str
```

Comment:
why do you save the password and not an hashed password?

Reply:
its the hashed password.
i call it just a password